_posts/2019-05-09-How-We-Release-Code.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How we release open source projects
+author: Per Ploug
+date: 2019-05-09
+banner:
+  image: connected.jpg
+---
+> Zalando currently have over 200 open source projects on Github and each month there are approximately 2 new projects proposed for publishing as open source.
+> Over the years we've refined our release process to ensure that it is easy and transparent to publish projects as open source and that we keep
+> our maintainers accountable for the well-being of the projects which they oversee.
```

Comment:
"""
Zalando has over 200 open source projects on Github, with about two new 
proposed projects a month. Over the years we've refined our process to 
ensure it is easy and transparent to publish projects as open source, 
and keep maintainers accountable for the well-being of their projects.
"""